tasks/main.yml

```diff
@@ -1,3 +1,19 @@
 ---
+- name: Get installed pip version
+  command: pip --version
+  register: pip_version_output
+  ignore_errors: yes
+  changed_when: false
+
+- name: Download get-pip.py
+  get_url: url=https://bootstrap.pypa.io/get-pip.py
+           dest=/tmp/get-pip.py
+
+# Install pip if it's not already installed, or if
+# the desired versions of pip aren't installed
+# The regular expression extracts '9.0' out of '9.0.*'
 - name: Install pip
-  apt: pkg=python-pip={{ pip_version }} state=present
+  command: python get-pip.py pip=={{ pip_version }}
```

Comment:
To be safe, I think that it would be good to quote this `command` because of the template variable.